certificate-service/domain/src/main/java/se/inera/intyg/certificateservice/domain/user/model/User.java

```diff
@@ -18,15 +18,16 @@
 @Builder
 public class User {
 
-  HsaId hsaId;
-  Name name;
-  Role role;
-  List<PaTitle> paTitles;
-  List<Speciality> specialities;
-  Blocked blocked;
-  Agreement agreement;
-  AllowCopy allowCopy;
-  AccessScope accessScope;
-  List<HealthCareProfessionalLicence> healthCareProfessionalLicence;
-  ResponsibleIssuer responsibleIssuer;
-}
+    HsaId hsaId;
+    Name name;
+    Role role;
+    List<PaTitle> paTitles;
+    List<Speciality> specialities;
+    Blocked blocked;
+    Agreement agreement;
+    AllowCopy allowCopy;
+    AccessScope accessScope;
+    List<HealthCareProfessionalLicence> healthCareProfessionalLicence;
+    ResponsibleIssuer responsibleIssuer;
+    SrsActive srs;
```

Comment:
Should we call the property srsActive to make it more clear? srs in itself doesn't say much what it is. Of course when you check the type SrsActive you understand... but it could make it more readable.